pkg/cmd/server.go

```diff
@@ -125,6 +127,9 @@ func serverCmd(cmd *cobra.Command, args []string) error {
 	go func() {
 		sigint := make(chan os.Signal, 1)
 		signal.Notify(sigint, os.Interrupt)
+		if runtime.GOOS != "windows" {
+			signal.Notify(sigint, syscall.SIGTERM)
```

Comment:
why you can't just use:
```
signal.Notify(sigint, os.Interrupt,syscall.SIGTERM )
```
instead? os one of this events will trigger shutdown.

Reply:
Windows has no concept of signals and so this wouldn't be portable AFAIK

Reply:
yes yes, I know, but it won't just work in that case?

Reply:
I mean you can just use both of them in one `signal.Notify` and one notification will be triggered based on current system. this is what I meant.

Reply:
hmm, I thought we would get errors when using `signal.Notify` on signals that don't exist on the runtime platform, but I just tested it on Windows and it works!